docker-compose-production.yml

```diff
@@ -0,0 +1,58 @@
+
+version: '3'
+services:
+  postgres:
+    image: postgres:11
+    ports:
+      - "5432:${DB_PORT}"
+    environment:
+      POSTGRES_USER: ${DB_USERNAME}
+      POSTGRES_PASSWORD: ${DB_PASSWORD}
+      POSTGRES_DB: ${DB_DATABASE}
+    networks:
+      - infra_postgres
+    volumes:
+      - pg-data:/var/lib/postgresql/data
+
+  server:
+    build:
+      dockerfile: "./Dockerfile"
+      context: "server/"
+    ports:
+      - "3000:${SERVER_PORT}"
+    environment:
+      SERVER_PORT: ${SERVER_PORT}
+      DB_TYPE: ${DB_TYPE}
+      DB_HOST: "postgres"
+      DB_PORT: ${DB_PORT}
+      DB_USERNAME: ${DB_USERNAME}
+      DB_PASSWORD: ${DB_PASSWORD}
+      DB_DATABASE: ${DB_DATABASE}
+    networks:
+      - infra_postgres
+      - infra_api
+    depends_on: ["postgres"]
+
+  client:
```

Comment:
this is nginx so isn't really a `client` because it doesn't perform HTTP requests towards the application. It's a `static` or an `assets` server?

Reply:
I mean, it serves the client to `server`? I don't mind renaming it, I think it should be called `frontend`, we just aren't using the `frontend`-`backend` terminology  /cc @diversemix

Reply:
it serves the client to the browser, but it's still a web server